pkgs/by-name/me/megapixels/search-for-postprocessors-in-NixOS-specific-global-location.patch

```diff
@@ -0,0 +1,45 @@
+--- a/src/process_pipeline.c
++++ b/src/process_pipeline.c
+@@ -171,7 +171,7 @@ mp_process_find_all_processors(GtkListStore *store)
+                         store, &iter, 0, buffer, 1, "(system) postprocess.sh", -1);
+         }
+ 
+-        // Check packaged /usr/share/megapixels/postprocess.sh
++        // Check packaged /nix/store/hash-megapixels-version/share/megapixels/postprocess.sh
```

Comment:
Patching the comments adds a bit of noise to the diff. I think it's easier to skim and understand when the only diffs are code changes.

Reply:
I removed that part of the patch.